source/common/http/header_map_impl.h

```diff
@@ -480,6 +483,14 @@ class RequestHeaderMapImpl final : public TypedHeaderMapImpl<RequestHeaderMap>,
   INLINE_REQ_RESP_STRING_HEADERS(DEFINE_INLINE_HEADER_STRING_FUNCS)
   INLINE_REQ_RESP_NUMERIC_HEADERS(DEFINE_INLINE_HEADER_NUMERIC_FUNCS)
 
+  absl::optional<absl::string_view> getTraceContext(absl::string_view key) const override;
+
+  void setTraceContext(absl::string_view key, absl::string_view val) override;
+
+  void setTraceContextReferenceKey(absl::string_view key, absl::string_view val) override;
+
+  void setTraceContextReference(absl::string_view key, absl::string_view val) override;
```

Comment:
```suggestion
  // Tracing::TraceContext
  absl::optional<absl::string_view> getTraceContext(absl::string_view key) const override;
  void setTraceContext(absl::string_view key, absl::string_view val) override;
  void setTraceContextReferenceKey(absl::string_view key, absl::string_view val) override;
  void setTraceContextReference(absl::string_view key, absl::string_view val) override;
```
Sorry, I forgot to add this suggestion last time. Usually we add a comment with a reference to the base class where the overridden methods come from.

Reply:
Yes. This is necessary. It was an oversight on my part. 👍